projects/aws/eks-anywhere/docker/linux/Dockerfile

```
@@ -5,6 +5,7 @@ ARG TARGETARCH
 ARG TARGETOS
 
 RUN yum install tar -y && \
+    yum install -y nmap iputils tar && \
```

tar is specified twice
```suggestion
RUN yum install -y nmap iputils tar && \
```